tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -1,18 +1,26 @@
 package test
 
 import (
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 	"os/exec"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+// Pining down version of Ruby Devkit to 2.5, 64 bit version
+// Ruby installation path kept default C:\
+var rubyExecutablePath = "C:\\Ruby25-x64\\bin\\ruby.exe"
+var rubyVersion = "2.5.7"
```

Comment:
why is this a global var?

Reply:
No need - I will make both `rubyVersion` and `rubyExecutablePath` variables local to `func TestRubyInstalled` in the upcoming commit